### ISSUE TITLE
Not able to set up the integration

### ISSUE THREAD

**Issue by mar-eid** [sindrebroch/ha-elvia#2]
## Version of the custom_component

Realeas v0.1.3

## Describe the bug
I'm able to install the integration, and metering point and API Key seems to be ok (got a Connection ok message when setting up the integration).

## Debug log

```text

Error setting up entry Elvia for elvia
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 357, in async_setup
    result = await component.async_setup_entry(hass, self)
  File "/config/custom_components/elvia/__init__.py", line 26, in async_setup_entry
    data = await api.meteringpoint()
  File "/config/custom_components/elvia/api.py", line 148, in meteringpoint
    return GridTariffCollection.from_dict(collection)
  File "/config/custom_components/elvia/models.py", line 423, in from_dict
    gridTariff=(GridTariff.from_dict(data["gridTariff"])),
  File "/config/custom_components/elvia/models.py", line 341, in from_dict
    tariffType=(TariffType.from_dict(data["tariffType"])),
TypeError: 'NoneType' object is not subscriptable

```

**Comment by mar-eid** [comment]
Update:

Completly my mistake, I Used the 2 character shorter customer number, rather than the meassuringpoint ID.

I figured when playing with Postman, and got a 4xx reply with a text describing that the meassuringpoint ID wasn't 11 characters. Maybe add a check on the to the config flow could be an idea?

**Comment by sindrebroch** [comment]
Great! 

Good suggestion, I will look into creating validation when setting up the integration.

**Comment by sindrebroch** [comment]
This should be fixed in release [0.1.4](https://github.com/sindrebroch/ha-elvia/releases/tag/0.1.4)
